### ISSUE TITLE
Duplicates in docs/functions (v4)

### ISSUE THREAD

**Issue by panva** [typedoc2md/typedoc-plugin-markdown#625]
### What package is the bug related to?

typedoc-plugin-markdown

### Describe the issue

Issue: Duplicate doc files are generated for some functions.

Reproduce with

```
gh repo clone panva/oauth4webapi -- --depth=1 --branch=typedoc-plugin-markdown-4
npm clean-install
npm run docs
```

See numerous markdown files in docs/functions duplicated with a `-1` suffix.

**Comment by tgreyuk** [comment]
Its because the functions are being tagged under 2 groups and are being duplicated as such. It is a bug and will be fixed. Thanks.

**Comment by tgreyuk** [comment]
fixed in typedoc-plugin-markdown@4.1.0
